index.html

```diff
@@ -265,8 +288,15 @@ <h3 id="other-deliverables"> Other Deliverables </h3>
             Recommendation, and which must be completed with an implementation
             report before transition to Proposed Recommendation. Additional
             tests may be added to the test suite at any stage of the
-            Recommendation track, and the maintenance of a implementation report
-            is encouraged. </dd>
+            Recommendation track, and the maintenance of an implementation
+            report is encouraged. </dd>
+          <dd> Additionally, this Working Group will seek to define an testing
+            API surface for the Presentation API and for the Remote Playback API
+            with a view to improve test automation, as well as to allow
+            developers to test their own presentations. Depending on progress,
+            this Working Group may choose to include the testing API in the
```

Comment:
Are we expecting to define separate testing API surface for the Presentation API and the Remote Playback API? If so, clearer would be to:

s/API/APIs/

If the testing API surface is shared across the Presentation API and the Remote Playback API, then the current singular form is fine if we s/them/it/ on L298.

Reply:
I guess the answer to that question may depend both on how we define the testing API and on where we define it. If we define it in the Presentation API spec but need something specific to the Remote Playback API, we'll end up with two different APIs. IF we define the testing API in a separate spec, it could cover both.